test/input/functor_multiple_local_vector.gt

```diff
@@ -0,0 +1,28 @@
+element Vertex end
+element Edge end
+const edges : edgeset{Edge}(Vertex, Vertex) = load ("../test/graphs/test.el");
+const vertices : vertexset{Vertex} = edges.getVertices();
+
+func initArray[a: vector{Vertex}(int), b: vector{Vertex}(int)](v: Vertex)
+    a[v] = 0;
+    b[v] = 0;
+end
+
+func addOne[a: vector{Vertex}(int), b: vector{Vertex}(int)](v: Vertex)
+    b[v] += 1;
+    a[v] += 2*b[v];
+end
+
+func main()
+
+    var simpleArray1 : vector{Vertex}(int) = new vector{Vertex}(int)();
```

Comment:
Need similar more-local-vector tests for "edgeset apply" and "edgeset applyUpdatePriority" cases. These tests should preferably have actual values in the vectors so that we can check the vectors are passed in correctly.